code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1481,12 +1487,13 @@
 
 /datum/chemical_reaction/slime/fire/on_reaction(var/datum/reagents/holder)
 	..()
-	sleep(50)
-	var/turf/location = get_turf(holder.my_atom.loc)
-	for(var/turf/simulated/floor/target_tile in range(0, location))
+	addtimer(CALLBACK(src, .proc/do_reaction, holder), 50)
+
+/datum/chemical_reaction/slime/fire/proc/do_reaction(var/datum/reagents/holder)
+	var/turf/location = get_turf(holder.my_atom)
+	for(var/turf/simulated/floor/target_tile in range(1, location))
 		target_tile.assume_gas("phoron", 25, 1400)
-		spawn (0)
-			target_tile.hotspot_expose(700, 400)
+		target_tile.hotspot_expose(700, 400)
```

Comment:
Have you checked if removing the spawn(0) does anything?

Reply:
![image](https://user-images.githubusercontent.com/22774890/78179424-76ae3700-7461-11ea-82ec-aba83c232529.png)